basic_credential/src/lib.rs

```diff
@@ -109,7 +109,7 @@ impl SignatureKeyPair {
         }
     }
 
-    fn id(&self) -> StorageId {
+    pub fn id(&self) -> StorageId {
```

Comment:
I didn't actually need this function to be public but figured if the struct was it should be. Totally cool with making this private atleast for my needs.